FNF_MissionTemplate.VR/description/cfgEventHandlers.hpp

```diff
@@ -6,10 +6,12 @@ class Extended_InitPost_EventHandlers {
   class Car {
     class PHX_NoBitchZone { init = "(_this select 0) allowCrewInImmobile true;"; };
     class PHX_DisableThermals { serverInit = "(_this select 0) disableTIEquipment true;"; };
+	class tnk_randomDamage { init = "((_this select 0) addMPEventHandler ['MPKilled', {params ['_unit', '_killer', '_instigator', '_useEffects']; if !(isServer) exitWith {}; [_unit] spawn PHX_fnc_handleDamage;}])"; };     
```

Comment:
Given that PHX_fnc_handleDamage only needs to be ran on the server, you can change 'init' here to 'serverInit 'and the MPEH to the standard "Killed" EH. This EH doesn't need to be global.

Reply:
Resolved with https://github.com/FridayNightFight/FNF/commit/dd8f3ca0d2588cafad474784d2b85beb3188c584